src/msw/listctrl.cpp

```diff
@@ -334,6 +334,11 @@ void wxListCtrl::MSWSetExListStyles()
         // it seems better to enable it by default than disable
         LVS_EX_HEADERDRAGDROP
     );
+
+    // If using the default background style with double buffering enabled then switch to system
+    // Using wxBG_STYLE_ERASE while will cause flicker despite double buffering
```

Comment:
I was thinking of checking for `IsDoubleBuffered()` in `wxWindowMSW::HandleEraseBkgnd()`, but I'm not 100% sure that it's not going to break anything...